.circleci/config.yml

```diff
@@ -0,0 +1,198 @@
+version: 2.1
+
+orbs:
+  android: circleci/android@0.2.1
+  gcp-cli: circleci/gcp-cli@2.1.0
+
+commands:
+
+  prepare-tests:
+    description: "prepares tests to run"
```

Comment:
I think this should be `Replace API_KEY` for consistency with https://github.com/RevenueCat/purchases-android/pull/215/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R166, and it's also more accurate

Reply:
note that unifying also helps if we actually want to build an orb to reuse these things